### PR TITLE
[RFR] Inist misc loaders

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,8 +24,11 @@ server:
         - mongo
     ports:
         - 3000:3000
+        - 8000:8000
+        - 8080:8080
```
Comment (djhi) [inline docker-compose.yml]: why ?

File: docker-compose.yml [hunk 2]
```diff
@@ -24,8 +24,11 @@ server:
...
 
 mongo:
     image: mongo:3.0.14
+    command: "--smallfiles"
```
Comment (djhi) [inline docker-compose.yml]: :+1:

File: src/api/loaders/json.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+
+ezs.use(ezsBasics);
+
+export default config => stream =>
+    stream
+    .pipe(ezs('JSONParse', { separator: '*', ...config }))
```
Comment (djhi) [inline src/api/loaders/json.js:8]: please indent

File: src/api/loaders/xml.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+
+ezs.use(ezsBasics);
+
+export default config => stream =>
+    stream
+        .pipe(ezs('XMLParse', { separator: [
+          '/RDF/*',
```
Comment (djhi) [inline src/api/loaders/xml.js]: please indent

**Review by djhi — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (touv) [reply docker-compose.yml]: oups, It's a patch to work on my pc

**Review by djhi — APPROVED** [review]

**Pull request merged**